schedule/yast/autoyast/autoyast_mig_spvm.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: autoyast_mig_spvm
```

Comment:
This is installation process and  prepared for migration test, how about name it as 'autoyast_install_spvm'?

Reply:
This schedule file includes 'patch_sle', it is not only about installation. So I think 'mig' is better.

Reply:
Updated